can_tools/scrapers/official/NM/nm_vaccine.py

```diff
@@ -0,0 +1,137 @@
+import pandas as pd
+import requests
+import us
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import StateDashboard
+from can_tools.scrapers.util import requests_retry_session
+
+
+class NewMexicoBase(StateDashboard):
+    state_fips = int(us.states.lookup("New Mexico").fips)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.session = requests_retry_session()
+
+
+class NewMexicoVaccineCounty(NewMexicoBase):
+    """
+    Fetch county level Covid-19 vaccination data from official state of New Mexico REST APIs
+    """
+
+    source = "https://cvvaccine.nmhealth.org/public-dashboard.html"
+    has_location = False
+    location_type = "county"
+
+    def fetch(self) -> requests.Response:
+        # Set url of downloadable dataset
+        url = "https://cvvaccine.nmhealth.org/api/GetCounties"
+        request = self.session.get(url)
+
+        if not request.ok:
+            message = f"Could not request data from {url}"
+            raise ValueError(message)
+
+        return request.json()
+
+    def normalize(self, data) -> pd.DataFrame:
+        # Read data into data frame
+        key = "data"
+        if key not in data:
+            raise ValueError(f"Expected to find {key} in JSON response")
+        df = pd.DataFrame(data[key])
+
+        # Determine what columns to keep
+        cols_to_keep = [
+            "county",
+            "date",
+            "modernaShipped",
+            "pfizerShipped",
+            "dosesAdministered",
+            "totalShipped",
+            "percentVaccinated",
+        ]
+
+        # Drop extraneous columns
+        df = df.loc[:, cols_to_keep]
+
+        # Rename columns
+        df = df.rename(columns={"date": "dt", "county": "location_name"})
+
+        # Convert date time column to a datetime
+        df = df.assign(dt=lambda x: pd.to_datetime(x["dt"]))
+
+        # Create dictionary for columns to map
+        crename = {
+            "modernaShipped": CMU(
+                category="moderna_vaccine_distributed",
+                measurement="cumulative",
+                unit="doses",
+            ),
+            "pfizerShipped": CMU(
+                category="pfizer_vaccine_distributed",
+                measurement="cumulative",
+                unit="doses",
+            ),
+            "dosesAdministered": CMU(
```

Comment:
The one thing that was kind of funny is that `dosesAdministered` seems to line up with the number on their dashboard that is used to denote "Individuals Partially Or Fully Vaccinated". I changed this to reflect what seems to be being reported rather than what they have chosen to name it... We should keep an eye on this because they might update it at some point

For example, in Catron (On February 5) the dashboard reports that 666 were partially or fully vaccinated and the `dosesAdministered` is 666.

Reply:
I guess I take `dosesAdministered` to mean X number of needles in arms.  I agree with the CMU (category="total_vaccine_doses_administered", measurement="cumulative", unit="doses").  I'm not sure if that was mine or yours post modification, but it looks good to me based on my current understanding.

Reply:
That was yours prior to modification -- I changed it to (`total_vaccine_initiated`, `cumulative`, `people`) because how how the number associated with that value lined up with values in the dashboard.

I definitely agree that `dosesAdministered` sounds more like (`total_vaccine_doses_administered`, `cumulative`, `doses`) which is why I wanted a second pair of eyes on this change :astonished: